Revamp list related BBEs (#3876)
ballerina-platform/ballerina-distribution: examples/filler-values-of-a-list/filler_values_of_a_list.md

# Filler values of a list

Lists can be initialized with the help of filler values. These are the default values that a list member will be initialized with if the user does not provide a value in the list constructor. This gives you the flexibility to assign the actual values later for lists.

::: code updating_a_list.bal :::

::: out updating_a_list.out :::

## Related links

> - [Filler values table - Specification](https://ballerina.io/spec/lang/master/#FillMember)

Author: @hasithaa @gimantha should we extract this table to a separate page or is it okay to point to the spec?

Reviewer: From users point of view, redirecting to a different link is not so user friendly. How about having it in the same place? Is it too much? @hasithaa

Reviewer 2: Agree with @gimantha, let's try to refer to a BBE.

Author: @hasithaa add in the same BBE, right?

Reviewer 2: yes

Author: Added via [3b9cbf6](https://github.com/ballerina-platform/ballerina-distribution/pull/3876/commits/3b9cbf645af18ae53b2a1e6def6e8f124c738305)

# Filler values of a list

> Lists can be initialized with the help of filler values. These are the default values that a list member will be initialized with if the user does not provide a value in the list constructor. This gives you the flexibility to assign the actual values later for lists.

Reviewer 3: ```suggestion
Lists can be initialized with the help of filler values. These are the default values that a list member will be initialized with if you do not provide a value in the list constructor. This gives you the flexibility to assign the actual values later for lists.
```
